src/main/java/com/kifiya/kobiri/controllers/BoutiqueController.java

```diff
@@ -15,23 +15,24 @@
 import javax.validation.Valid;
 
 @Controller
+@RequestMapping("/admin")
 public class BoutiqueController {
     @Autowired
     private BoutiqueService boutiqueService;
 
-    @RequestMapping(value = "boutiques", method = RequestMethod.GET)
+    @RequestMapping(value = "/boutiques", method = RequestMethod.GET)
     public String obtenir(Model model){
         model.addAttribute("boutiques", boutiqueService.findAll());
         model.addAttribute("boutique",new Boutique());
         return "boutique/boutique";
     }
 
-    @RequestMapping(value = "boutiques", method = RequestMethod.POST)
+    @RequestMapping(value = "/boutiques", method = RequestMethod.POST)
     public String ajouter(@Valid @ModelAttribute("boutique") Boutique boutique,
                               BindingResult result, HttpServletRequest request, Model model){
         if (result.hasErrors()) {
             //AJouter le modele error de saissie
-            return "boutique/boutique";
+            return "error/400";
```

Comment:
pourquoi le fichier s'appelle 400 ?
```suggestion
            return "error-400";
```